agent/src/main/kotlin/org/jetbrains/teamcity/vault/agent/VaultParameter.kt

```diff
@@ -28,5 +28,5 @@ data class VaultParameter constructor(val vaultPath: String, val jsonPath: Strin
         }
     }
 
-    val full: String get() = (if (jsonPath == null) vaultPath else vaultPath + "!/" + jsonPath).ensureHasPrefix("/")
+    val full: String get() = (if (jsonPath == null) vaultPath else "$vaultPath!/$jsonPath").ensureHasPrefix("/")
```

Comment:
It could be more readable but unnecessary for this PR

Reply:
reverted